### PR TITLE
2253 refactor/api v3 mbu

### PR DESCRIPTION
**Pull request by aviveiros11**
#### What's this PR do? 
Moves and refactors v2 meters endpoints into v3
#### How should this be manually tested?
Test the endpoints on the swagger page
#### What are the relevant tickets?
#2253

### CODE REVIEW

**Comment by aviveiros11** [comment]
@macintoshpie turns out that trying to override the decorator for the second endpoint `create_from_pm_import` did not work.  Trying to add a `request_body` kept throwing the error of `form request body cannot be a Schema`.  Gonna look more into overriding and assigning parser classes.

**Comment by macintoshpie** [comment]
> @macintoshpie turns out that trying to override the decorator for the second endpoint `create_from_pm_import` did not work. Trying to add a `request_body` kept throwing the error of `form request body cannot be a Schema`. Gonna look more into overriding and assigning parser classes.

Not sure if you got it sorted out, but I found this here which describes how to fix it: https://github.com/axnsan12/drf-yasg/issues/386#issuecomment-553882090

**Comment by aviveiros11** [comment]
Thank you @macintoshpie!

File: seed/views/v3/uploads.py [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+            },
+            required=['import_record_id', 'properties'],
+            description='An object containing meta data for a property'
+        )
+    )
+    @has_perm_class('can_modify_data')
+    @api_endpoint_class
+    @ajax_request_class
+    @action(detail=False, methods=['POST'], parser_classes=(JSONParser,))
+    def create_from_pm_import(self, request):
```
Comment (macintoshpie) [inline seed/views/v3/uploads.py:181]: Looking for feedback on this. Not confident I've tested this well.

**Pull request merged**